src/asm2wasm.h

```diff
@@ -664,6 +664,10 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
     }, debug, false /* do not validate globally yet */);
   }
 
+  // if we see no function tables in the processing below, then the table still exists and has size 0
+
+  wasm.table.initial = wasm.table.max = 0;
```

Comment:
Ah i see. in that case is it even necessary to have this line then? This line doesn't cause the table to exist, and the table constructor sets both of the limits to 0 anyway?